apps/dapp/src/providers/StakingProvider.tsx

```diff
@@ -1,39 +1,14 @@
-import {
-  useState,
-  createContext,
-  useContext,
-  PropsWithChildren,
-} from 'react';
-import { BigNumber, ethers, Signer } from 'ethers';
+import { useState, createContext, useContext, PropsWithChildren } from 'react';
```

Comment:
looks like my linter might have a different max length than what this file was previously linted as, sorry for the many changes 🙇